es_test_data.py

```diff
@@ -88,10 +99,22 @@ def get_data_for_format(format):
         max = now + 30 * per_day if len(split_f) < 4 else int(split_f[3])
         return_val = int(random.randrange(min, max) * 1000)
 
+
+    elif field_type == "ts2":
```

Comment:
Want to call this `tstxt` instead? `ts2` works as well but could be a bit more descriptive.